### ISSUE TITLE
Java code

### ISSUE THREAD

**Issue by HrishiGodina** [atom-community/atom-script#1369]
I'm not able to run java code in atom editor even though "script" package is installed. 
```
" java - Work.java:7
Error: Could not find or load main class .home.hrishi99.Work
[Finished in 2.384s] "
```
I'm getting this error. 

-------
Linix machine

**Comment by OktayGardener** [comment]
You need to set up the package.

if your folder looks like:
dev/work/test.java

then you should add to your file:
`@package work;`

**Comment by aminya** [comment]
Should be fixed in #2470. Let me know if it is not.
